.github/Pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+## Related Issuse
+  - Info about Issue or bug
+
+Closes: #[issue number that will be closed through this PR]
+
+#### Describe the changes you've made
+A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!
+
+## Type of change
+
+<!--
+Example how to mark a checkbox:-
+- [x] My code follows the code style of this project.
+-->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Code style update (formatting, local variables)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

```

Comment:
The test cases will be recorded and replayed via Keploy since we're testing Keploy with Keploy.
So we can just ask if there are any test cases added and how's it affecting the coverage. :)